### ISSUE TITLE
Expand CLI

### ISSUE THREAD

**Issue by mathisonian** [idyll-lang/idyll#264]
It would be nice to add additional commands to the CLI:

```sh
# generate a new Idyll project folder
$ idyll init 

# build the project for deployment to the web,
# including copying assets, etc.
$ idyll build

# publish the project to the web
$ idyll publish
```

**Comment by mathisonian** [comment]
@andrewosh when we get `idyll init` and `idyll build` working, that seems like a good place to mark 3.0.

**Comment by mathisonian** [comment]
Fixed by #333
